src/formio/components/Children/Children.scss

```
@@ -0,0 +1,22 @@
+.openforms-children {
+  padding: 1em;
+
+  @media (max-width: 768px) {
+    overflow: auto;
+    overflow-wrap: break-word;
+    -webkit-overflow-scrolling: touch;
+
+    &__actions {
+      display: inline-flex;
+    }
+
+    .utrecht-table__header-cell {
+      padding-inline-end: 4rem;
+      white-space: nowrap;
+    }
+
+    .utrecht-table__cell {
+      padding-inline-end: 4rem;
+    }
```

there are design tokens for the inline paddings though:


```suggestion
    --utrecht-table-cell-padding-inline-end: 20px;
    
    .utrecht-table__header-cell {
      white-space: nowrap;
    }
```
the same token is used for both regular and header cells